src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -690,6 +947,21 @@ def set_errors_enabled(self, enabled: bool):
     async def interrupt_bot(self):
         await self.push_frame(BotInterruptionFrame(), FrameDirection.UPSTREAM)
 
+    async def send_server_message(self, data: Any):
```

Comment:
I see an example of using `send_server_response()` but no examples of using `send_server_message()`